docs/tutorial/graph_partition.ipynb

```diff
@@ -910,14 +931,18 @@
    ],
    "source": [
     "sampleset = qaoa_converter.decode(quri_transpiler, (qp_result, quri_circuit.qubit_count))\n",
-    "energies = []\n",
-    "num_occurrences = []\n",
-    "for sample in sampleset.feasibles():\n",
-    "    energies.append(sample.eval.objective)\n",
-    "    num_occurrences.append(sample.num_occurrences)\n",
-    "plt.bar(energies, num_occurrences, width=0.5)\n",
-    "plt.ylabel(\"Frequency\")\n",
-    "plt.xlabel(\"objective\")\n",
+    "\n",
+    "# Initialize a dictionary to accumulate occurrences for each energy value\n",
+    "frequencies = defaultdict(int)\n",
+    "\n",
+    "# Define the precision to which you want to round the energy values\n",
+    "for sample in sampleset:\n",
+    "    energy = round(sample.eval.objective, ndigits = 3)  \n",
+    "    frequencies[energy] += sample.num_occurrences\n",
+    "\n",
+    "plt.bar(frequencies.keys(), frequencies.values(), width=0.05)\n",
```

Comment:
I think in this problem `width = 0.05` is too narrow, so can you change `width = 0.5`?

Reply:
@Jacomichi Done